packages/components/src/components/input/input.jsx

```diff
@@ -28,6 +30,7 @@ const Input = (
         e.target.value = input_value;
         props.onChange(e);
     };
+    const dataLpignore = lpignore || (type === 'password' ? false : true);
```

Comment:
by default, it would be the option that a user passes to it. if not, it checks the type of input